inc/modules/export/xhtml/class-xhtml11.php

```diff
@@ -1003,10 +1014,14 @@ protected function echoTitle( $book_contents, $metadata ) {
 			printf( '<h1 class="title">%s</h1>', get_bloginfo( 'name' ) );
 			printf( '<h2 class="subtitle">%s</h2>', ( isset( $metadata['pb_subtitle'] ) ) ? $metadata['pb_subtitle'] : '' );
 			if ( isset( $metadata['pb_authors'] ) ) {
-				printf( '<h3 class="author">%s</h3>', $metadata['pb_authors'] );
+				foreach ( $metadata['pb_authors'] as $author ) {
+					printf( '<h3 class="author">%s</h3>', $author['name'] );
```

Comment:
This is going to result in several H3 elements, rather than one H3 element with several authors. Probably not our desired behaviour.

Reply:
Yes, it produces multiple H3 elements in the exported file. What would be the output for multiple authors and contributors? Maybe an "Authors" title (h3) with a list (ul)? and same for contributors.

Reply:
This function displays the text that appears on the 'title page' for export files. Right now we display a series of heading elements -- but this is part of what should be refactored when we improve our exports -- post of these should probably be printed as paragraphs with specific classes that are styled to display as desired. For now, we can simply print a single h3 element with class authors and then the names of each author with commas between them and an 'and' before the last authors name.